### PR TITLE
docs(update): added deprecated tag to brownie and Cyfrin Updraft

### PR DESCRIPTION
**Pull request by cromewar**
Updated Docs:

- Added the deprecated tag to Brownie, as it is not maintained anymore.
- Added Cyfrin Updraft to general resources as it is a great way to learn Solidity Development.
- Added Aderyn to Solidity tools because is a nice Static analyzer tool for Solity.

### CODE REVIEW

**Review by mehtavishwa30 — COMMENTED** [review]
Hey @cromewar!

Thank you for your contribution. There is currently an ongoing update that will add appropriate additional resources and bring the existing items in the list up-to-date in a PR (see: https://github.com/ethereum/solidity/pull/15076).

We will be reviewing and combining smaller individual PRs like yours in a single change. :)

**Comment by PatrickAlphaC** [comment]
Hello! Any update on this PR?

File: docs/resources.rst [hunk 1]
```diff
@@ -15,11 +15,12 @@ General Resources
 * `Awesome Solidity <https://github.com/bkrem/awesome-solidity>`_
 * `Solidity by Example <https://solidity-by-example.org/>`_
 * `Solidity Documentation Community Translations <https://github.com/solidity-docs>`_
+* `Cyfrin Updraft - The Leading Educative Platform for Blockchain Developers <https://updraft.cyfrin.io/>`_
```
Comment (ibrahimkhled) [inline docs/resources.rst:18]: docs/resources.rst

**Review by ibrahimkhled — COMMENTED** [review]
__

File: docs/resources.rst (hunk 1 above)
Comment (PatrickAlphaC) [reply docs/resources.rst:18]: Oh nice! Thank you

**Comment by PatrickAlphaC** [comment]
@cameel should we make a new PR for this? Or is it not wanted.

**Comment by cameel** [comment]
It was closed along with a few others that became redundant after #15076. So that's a question to @mehtavishwa30.

**Comment by mehtavishwa30** [comment]
Hey @PatrickAlphaC! Apologies for the missing clarification earlier. We had several external PRs to update the resources page that were accommodated in #15076.

From @cromewar's PR, 
- I added a note to address the deprecation of Brownie as suggested
- Also added in Aderyn (great suggestion!)
- I left out Updraft because IMO, it doesn't fit under our general resources section, as all resources focused on education continue to live on the [ethereum.org](http://ethereum.org/) developer portal instead.

I hope this is helpful! Lmk if you have thoughts. :)

**Comment by PatrickAlphaC** [comment]
Got it, thank you!

**Pull request closed without merging**